test/e2e/quadlet_test.go

```diff
@@ -172,6 +172,24 @@ func (t *quadletTestcase) assertKeyIsRegex(args []string, unit *parser.UnitFile)
 	return true
 }
 
+func (t *quadletTestcase) assertLastKeyIsRegex(args []string, unit *parser.UnitFile) bool {
+	Expect(len(args)).To(BeNumerically(">=", 3))
+	group := args[0]
+	key := args[1]
+	regex := args[2]
+
+	value, ok := unit.LookupLast(group, key)
+	if !ok {
+		return false
+	}
+
+	ok, err := regexp.MatchString(regex, value)
+	if err != nil {
+		return ok
+	}
+	return false
```

Comment:
I've found the error.
```suggestion
	matched, err := regexp.MatchString(regex, value)
	if err != nil || !matched {
		return false
	}
	return true
```